src/web/common/Countdown.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+function Countdown({ callback, className, timeInSeconds, paused }) {
+  const [currentTime, setCurrentTime] = useState('00:00');
+  let logInterval;
+  let duration = timeInSeconds;
+
+  const startCountdown = () => {
+    let minutes;
+    let seconds;
+
+    logInterval = setInterval(() => {
+      minutes = parseInt(duration / 60, 10);
```

Comment:
I'm wondering if it wouldn't be better to use a [Moment.js duration](https://momentjs.com/docs/#/durations/) to calculate and display the remaining time.